_lessons/design-thinking/design-reports.md

```diff
@@ -55,7 +55,7 @@ Suggested structure
 : What would you do differently next time?
 
 6. Appendix
-In order to not overload the report, some elements might be added in the appendix such as
+In order to not overload the report, some elements might be added in the appendix such as:
```

Comment:
I think it would be better to use bullet points instead of a definition list here and above. In markdown, this would look like:

```md
5. **Part 3: Learnings & Next Steps**
    - Explain next steps: what features you would add / change
    - Learnings: what did you learn? what were you biggest challenges?
    - Try to focus on a personal point of view
    - What would you do differently next time?
```

5. **Part 3: Learnings & Next Steps**
    - Explain next steps: what features you would add / change
    - Learnings: what did you learn? what were you biggest challenges?
    - Try to focus on a personal point of view
    - What would you do differently next time?